R/0_imports.R

```diff
@@ -14,22 +14,6 @@ utils::globalVariables(
 
 # ------------------------------------------------------------------------------
 
-#' @importFrom tibble tbl_sum
-#' @export
-tibble::tbl_sum
-
-#' @importFrom tibble type_sum
-#' @export
-tibble::type_sum
-
-#' @importFrom tibble size_sum
-#' @export
-tibble::size_sum
-
-#' @importFrom tibble obj_sum
-#' @export
-tibble::obj_sum
-
```

Comment:
You need `@importFrom` for each of these (to be able to add S3 methods), but you don't need `@export`. So this would probably suffice:

```r
#' @importFrom tibble tbl_sum, type_sum, size_sum, obj_sum
```

Reply:
I'm up-to-date locally and have the same but It's not showing up in the PR. 

![image](https://user-images.githubusercontent.com/5731043/126555117-97405525-70ba-4393-9c7f-6278f8dadecf.png)

I'm going to `pr_fetch()` it and see what's what. ¯\_(ツ)_/¯

Reply:
Crap. I did it on master 😢  I'll update from that.

Reply:
I'm up-to-date in this PR so I think that it's fine to merge. Time to call it a day I think 🙄